web/settings/modulconfiglp.php

```diff
@@ -4509,6 +4520,8 @@ function display_socmodul1() {
 							hideSection('#socevcclp2');
 							hideSection('#socmkialp2');
 							hideSection('#socoldevccwarninglp2');
+							hideSection('#socmvwidinfo');
```

Comment:
Die Zeile gehört in den Bereich für Ladepunkt 1.

Reply:
Hallo Lutz, sollte jetzt passen...